docs/api/model_selection/learning_curve.rst

```diff
@@ -93,23 +85,25 @@ Clustering
 
 Learning curves also work for clustering models and can use metrics that specify the shape or organization of clusters such as silhouette scores or density scores. If the membership is known in advance, then rand scores can be used to compare clustering performance as shown below:
 
-.. code:: python
+.. plot::
+    :context: close-figs
+    :alt: Learning Curve on clustering models
 
     from sklearn.cluster import KMeans
     from sklearn.datasets import make_blobs
 
-    # Create a dataset of blobs
-    X, y = make_blobs(n_samples=1000, centers=5)
-
-    viz = LearningCurve(
-        KMeans(), train_sizes=sizes, scoring="adjusted_rand_score"
-    )
+    # Generate synthetic dataset with 5 random clusters
+    X, y = make_blobs(n_samples=1000, centers=5, random_state=42)
```

Comment:
Great use of random_state, I think it is good to have figures that users can replicate